hugo/layouts/_partials/head/styles.html

```diff
@@ -48,14 +48,6 @@
     height: 24px;
   }
 
-  .frozen-row {
-    background: rgba(0, 0, 0, 0.05);
-  }
-
-  .frozen-row-bottom-border {
-    border-bottom: 1px solid rgb(182, 182, 182);
-  }
-
```

Comment:
Now that we're not implementing our own custom solution for freezing header rows, we don't need these classes anymore.